src/Lottos.js

```diff
@@ -0,0 +1,14 @@
+class Lottos {
+  #lottoNumbers;
+
+  constructor(lottoNumbers) {
+    this.#lottoNumbers = lottoNumbers;
+  }
+
+  getLottoNumbers = () => {
+    const lottoList = this.#lottoNumbers.map((lotto) => lotto.getLottoNumber());
+    return lottoList;
```

Comment:
```suggestion
  getLottoNumbers = () => {
    return this.#lottoNumbers.map((lotto) => lotto.getLottoNumber());
```
저는 개인적으로 이렇게 축약해도 맥락을 파악할 수 있을 것 같은데 종빈님은 축약하지 않으신 이유가 있으신가요??

Reply:
아뇨..! 축약할 수 있는데 제가 놓친 것 같아요🥲